.husky/set-node.sh

```
@@ -171,79 +171,71 @@ setup_pnpm() {
     log_error "pnpm 패키지 매니저가 아닙니다!"
     log_warning "pnpm을 사용하세요: > pnpm install <"
-    exec $SHELL
+    exec "$SHELL"
     exit 1
   fi
 }
 
 # 사용자에게 패키지 매니저를 선택하도록 하는 함수
 select_package_manager() {
-  local available_managers=()
+  local available_managers=""
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)